### DISCUSSION TITLE
whisper.cpp in SubtitleEdit

### DISCUSSION THREAD

**Discussion by niksedk** [ggml-org/whisper.cpp#159] [Show and tell]
I've added whisper.cpp to my small utility for subtitles, Subtitle Edit - in version 3.6.10

![image](https://user-images.githubusercontent.com/1261331/202921272-cea57300-ec27-4f7d-bb43-1870ceaf6950.png)

https://github.com/SubtitleEdit/subtitleedit/releases

So, thx for whisper.cpp :)
(Wishing more accurate timestamps for Christmas)

Platform: Mainly Windows, a little Linux

**Comment by Baenwort** [comment]
This, especially with timestamp improvements, will mean it is a lot easier to make quality subtitles for things that often don't get them like fanedits and indy films.

**Comment by manumaan** [comment]
@niksedk long time fan! Thanks for making the great software. I actually used your app to create a subtitle for a fanedit movie recently.
